### PR TITLE
added cloud computing course in hindi language

### PR DESCRIPTION
**Pull request by Shorya18**
## What does this PR do?

Added resources for cloud computing in Hindi Language

### Description
I have added some sources to learn cloud computing easily.
### Why is this valuable (or not)?
this is valuable as it is an important topic which was not present in the list.
### How do we know it's really free?
its on youtube and thus we do not need to pay for it also i have already checked it.
### For book lists, is it a book? For course lists, is it a course? etc.
it is a course.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
need to add category to index. 
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. The AWS resources could move here.

**Comment by Shorya18** [comment]
hey I have moved the AWS resources as told by you to cloud computing category and also made the necessary changes in index.

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
 * [Artificial Intelligence](#artificial-intelligence)
 * [Assembly](#assembly)
-* [AWS](#aws)
 * [Bash and Shell](#bash-and-shell)
 * [Blockchain](#blockchain)
 * [C](#c)
 * [C#](#csharp)
     * [ASP.NET](#asp.net)
 * [C++](#cpp)
+* [cloud computing](#Cloud)
```
Comment (eshellman) [inline courses/free-courses-hi.md:15]: this link won't work

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -192,6 +192,7 @@
 * [ASP.NET Course 2023](https://www.youtube.com/playlist?list=PLMoluEXvWXK6Q1h-5vVX4tzX7-O2FdgZA) - Coder Baba
 
 
+
```
Comment (eshellman) [inline courses/free-courses-hi.md:195]:
```suggestion
```

**Comment by eshellman** [comment]
already merged

**Pull request closed without merging**